pkg/runtime/wfengine/backend.go

```diff
@@ -25,52 +25,99 @@ import (
 
 	"github.com/dapr/dapr/pkg/actors"
 	invokev1 "github.com/dapr/dapr/pkg/messaging/v1"
+	"github.com/dapr/dapr/utils"
 )
 
-// workflowScheduler is an interface for pushing work items into the backend
-type workflowScheduler interface {
-	ScheduleWorkflow(ctx context.Context, wi *backend.OrchestrationWorkItem) error
-	ScheduleActivity(ctx context.Context, wi *backend.ActivityWorkItem) error
+// actorsBackendConfig is the configuration for the workflow engine's actors backend
+type actorsBackendConfig struct {
```

Comment:
This struct and the functions that follow were moved from wfengine.go to here since they are specific to the workflow backend. No changes were made to these.